lisa/utils.py

```diff
@@ -135,17 +144,47 @@ def _init_yaml(cls):
         yaml = cls._yaml
         # If allow_unicode=True, true unicode characters will be written to the
         # file instead of being replaced by escape sequence.
-        yaml.allow_unicode = (cls.YAML_ENCODING == 'utf-8')
+        yaml.allow_unicode = ('utf' in cls.YAML_ENCODING)
         yaml.default_flow_style = False
         yaml.indent = 4
         yaml.Constructor.add_constructor('!include', cls._yaml_include_constructor)
         yaml.Constructor.add_constructor('!var', cls._yaml_var_constructor)
         yaml.Constructor.add_multi_constructor('!call:', cls._yaml_call_constructor)
+
+        # Replace unknown tags by a placeholder object containing the data This
```

Comment:
Missing comma? Also, out of curiosity, how did you hit this issue? I don't think I've ever hit that.

Reply:
It happens if you load a YAML file containing tags for classes that have not been created yet (you have not imported the module). That comes from the fact that if you call cls._yaml.register_class() on Serializable classes, the default tag will be cls.__qualname__, instead of being a full dotted path including the module name. As a result, the tag is more stable against refactoring, but it is also not possible for ruamel.yaml to find the class if it has not been registered. 

It should not be a big problem since if you haven't imported the corresponding module, chances are you don't care about these objects. Classes for objects used internally by classes that matters are usually imported indirectly, so it's fine. Ultimately, that also allows to inspect the content of a YAML file that cannot be completely parsed instead of a global failure at load time, so it improves robustness

The main reason I call register_class() is because it's the only way to take into account the optional class attribute yaml_tag, and that is desirable to maintain a stable set of tags in case we want to rename the class.